main.go

```diff
@@ -62,7 +63,7 @@ func mainWithError() (err error) {
 				panic(fmt.Sprintf("%#v\n", microerror.Maskf(err, "service.New")))
 			}
 
-			go newService.Boot()
+			go newService.Boot(context.Background())
```

Comment:
I think the context should be defined at the top of `mainWithError` rather than inline. As we might add more logic there later.